packages/in_app_purchase/pubspec.yaml

```diff
@@ -1,30 +1,26 @@
 name: in_app_purchase
 description: A Flutter plugin for in-app purchases. Exposes APIs for making in-app purchases through the App Store and Google Play.
 homepage: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase
-version: 0.3.5+2
+version: 0.4.0-nullsafety.0
```

Comment:
Looks like you can just do 0.4.0 and avoid a second pass to publish this as stable later.

Reply:
done